tests/cartservice/cartservice.tests.csproj

```diff
@@ -1,19 +1,19 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp5</TargetFramework>
+    <TargetFramework>net5.0</TargetFramework>
 
     <IsPackable>false</IsPackable>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Grpc.Net.Client" Version="2.32.0" />
-    <PackageReference Include="Grpc.Tools" Version="2.23.0" />
-    <PackageReference Include="Google.Protobuf" Version="3.12.3" />
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.7.0" />
-    <PackageReference Include="xunit" Version="2.3.1" />
-    <PackageReference Include="xunit.runner.visualstudio" Version="2.3.1" />
-    <DotNetCliToolReference Include="dotnet-xunit" Version="2.3.1" />
+    <PackageReference Include="Grpc.Net.Client" Version="2.34.0" />
+    <PackageReference Include="Grpc.Tools" Version="2.34.0" />
+    <PackageReference Include="Google.Protobuf" Version="3.14.0" />
```

Comment:
Did you test that tests for cart service pass with your code?

Reply:
I didn't, but I'm on it and will report back once done. In the meantime, I have hard time to get the `tests/carservice/tests.yaml` working, is it working on `master`?

Reply:
I was able to successfully run the tests, but I needed to apply a fix https://github.com/GoogleCloudPlatform/cloud-ops-sandbox/pull/612/commits/7c8c7479e717e8953552a23fdbce95c7a31d1345.